codalab/objects/permission.py

```diff
@@ -152,9 +154,14 @@ def permission_str(permission):
     raise UsageError("Invalid permission: %s" % permission)
 
 # [{'group_name':'a', 'permission:1}, {'group_name':'b', 'permission':2}] => 'a:read,b:all'
-def group_permissions_str(group_permissions):
+def group_permissions_str(group_permissions, use_rest=False):
```

Comment:
Sorry about this, another rebase/`git add --interactive` issue! This should be with the next PR not this one. I'll move it.